src/mmrelay/plugins/mesh_beacon_plugin.py

```
@@ -0,0 +1,432 @@
+            self.logger.error("Mesh Beacon configuration not applied: %s", exc)
+        except Exception:
+            self.logger.exception("Failed to configure firmware Mesh Beacon module")
+        else:
+            if changed:
+                self.logger.info("Applied firmware Mesh Beacon configuration")
+            else:
+                self.logger.debug(
+                    "Firmware Mesh Beacon configuration is already current"
+                )
+
+    def configure_firmware(self, interface: Any) -> bool:
```

Ruff (0.16.3)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `interface`

(ANN401)